src/Helpers.php

```diff
@@ -170,7 +170,7 @@ public static function mapAuthorityByClass(array $authorities)
      *
      * @param  iterable  $items
      * @param  callable  $callback
-     * @return static
+     * @return \Illuminate\Support\Collection[]|\Illuminate\Support\Collection
```

Comment:
This method only has a single return type: a collection (with two arrays inside).

`\Illuminate\Support\Collection[]` means an array of collections. This method never returns a collection.

Reply:
Indeed it returns a `Collection` instance. With two `Collection`s inside.

Reply:
Again:

1. There's only a single type being returned from this method. You have the return type as variant.

2. A collection of collections is not the same as an array of collections.

Just remove the first variant.

Reply:
Ok, I have removed it.